includes/class-wc-payments-onboarding-service.php

```diff
@@ -718,7 +720,7 @@ public function init_test_drive_account( string $country, array $capabilities =
 		// the WooPay capability value from the request.
 		$should_enable_woopay = $this->should_enable_woopay(
 			filter_var( $onboarding_data['woopay_enabled_by_default'] ?? false, FILTER_VALIDATE_BOOLEAN ),
-			$this->get_capabilities_from_request()
+			$capabilities
```

Comment:
if `$this->get_capabilities_from_request` is unused, we should remove it!

Reply:
Although I initially intended to remove this code, I've kept it temporarily since it's still referenced within the `maybe_handle_onboarding` method, and it will be cleaned up when that method is eventually removed.